### PR TITLE
feat: add persistant storage methods to Python SDK

### PR DESCRIPTION
**Pull request by kafkaphoenix**
## Description

- Add persistent storage methods to the Python SDK
- Update demo examples
- Update validation logic for each SDK

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: demo/cronjob-trigger-py/app.yaml [hunk 1]
```diff
@@ -13,6 +13,14 @@ nats:
...
   object_store: ephemeral-storage
 
+minio:
+  endpoint: "localhost:9000"
+  access_key: "minio"
+  secret_key: "minio123"
+  region: "us-east-1"
+  secure: false
+  bucket: "test-bucket"
+
```
Comment (dfinteligenz) [inline demo/cronjob-trigger-py/app.yaml:23]: Just a couple of changes @kafkaphoenix.

We will finally be using the next configuration (some naming changes, and remove the region):
```yaml
minio:
  url: "localhost:9000"
  access_key_id: minioadmin
  access_key_secret: minioadmin
  use_ssl: false
  bucket: test-bucket
````

**Review by dfinteligenz — CHANGES_REQUESTED** [review]
We need to do a small change, check the comment above.

File: demo/cronjob-trigger-py/app.yaml (hunk 1 above)
Comment (kafkaphoenix) [reply demo/cronjob-trigger-py/app.yaml:23]: going to update and all of them and the validators

Comment (kafkaphoenix) [reply demo/cronjob-trigger-py/app.yaml]: done

**Review by dfinteligenz — APPROVED** [review]

**Pull request merged**